data/remnant/remnant ships.txt

```diff
@@ -1183,33 +1188,30 @@ ship "Peregrine"
 		licenses
 			"Remnant Capital"
 		"cost" 41627000
-		"shields" 16400
-		"hull" 10600
+		"shields" 15400
+		"hull" 8600
 		"required crew" 9
 		"bunks" 25
 		"mass" 350
 		"drag" 4.3
 		"heat dissipation" 0.85
-		"fuel capacity" 1100
+		"fuel capacity" 900
 		"ramscoop" 3.25
-		"cargo space" 59
+		"cargo space" 49
 		"outfit space" 552
 		"weapon capacity" 220
 		"engine capacity" 171
-		"shield generation" 4.9
+		"shield generation" 3.9
 		"shield energy" 3.0
-		"hull repair rate" 2.5
+		"hull repair rate" 1.5
 		"hull energy" 0.90
-		"cloak" .04
-		"cloaking energy" 2.5
+		"cloak" .03
+		"cloaking energy" 3
 		"cloaking fuel" .05
```

Comment:
```suggestion
		"cloak" .03
		"cloaking energy" 4
		"cloaking fuel" .075
```
Adjust fuel cost to .075 (4.5/sec) rather than .05 (3/sec), in line with and for the same reasons as on the Merganser. Peregrine has more built-in ramscoop and more outfit space to fit more fuel outfits, so is much less affected.
Cloaking energy is largely irrelevant unless it's much higher than these numbers anyways, and this change is tiny, so I don't see any issue leaving it unchanged. Cloaking energy isn't significant either way unless it's much higher than this; this suggested change puts it at the same level as the Merganser (because the two are from the same generation of ships). You could put both down to 3, or to 2.5, and it wouldn't really make a difference balance-wise, but I do think thematically they ought to match.